include/albatross/src/core/parameter_handling_mixin.hpp

```diff
@@ -142,13 +142,32 @@ class ParameterHandlingMixin {
     }
   }
 
+  void set_params_if_exists(const ParameterStore &params) {
```

Comment:
What's the use case for these functions? When would the ParameterStore not have the exact same parameters as the model being set?

Reply:
I end up using this where I have one model with a given covariance function and another with a subset covariance function and I want the smaller model to take on the parameters of the parent.  I don't see this is a very common pattern, but it beats re-writting the `if(contains){set}` pattern in multiple places.